internal/runners/scripts/edit.go

```diff
@@ -94,7 +94,7 @@ func createScriptFile(script *project.Script, expand bool) (*scriptfile.ScriptFi
 		scriptBlock = script.Value()
 	}
 
-	f, fail := scriptfile.NewAsSource(script.LanguageSafe(), script.Name(), scriptBlock)
+	f, fail := scriptfile.NewAsSource(script.LanguageSafe()[0], script.Name(), scriptBlock)
```

Comment:
Do I even have to say it? :p

Reply:
lol yeah I went back and forth on this one as it shouldn't be possible for `LanguageSafe` to return an empty slice. Updated the usage to fallback to the default scripting languages